docs/sketch-specification.md

```diff
@@ -79,6 +79,14 @@ The `included_libs` key defines the library versions the Arduino Web Editor uses
 Arduino Web Editor specific because all versions of all the Library Manager libraries are pre-installed in Arduino Web
 Editor, while only one version of each library may be installed when using the other Arduino development software.
 
+### Sketch project file
+
+This is an optional file named `sketch.yaml`, located in the root folder of the sketch.
+
+Inside `sketch.yaml` the user can define one or more "profiles": each profile is a description of all the resources
+needed to build the sketch (platform and libraries each pinned to a specific version). For more information see the
+[sketch project file](sketch-project-file.md) documentation.
```

Comment:
```suggestion
Inside the sketch project file the user can define one or more "profiles": each profile is a description of all the
resources needed to build the sketch (platform and libraries each pinned to a specific version).

For more information see the [sketch project file](sketch-project-file.md) documentation.
```

- Use standard "sketch project file" term.
- Move reference link to dedicated paragraph for readability